### PR TITLE
Added SQL to backend development languages

### PR DESCRIPTION
**Pull request by lokytech5**
Added sql roadmap to backend languages. please compare and let me know. thanks

### CODE REVIEW

**Comment by lokytech5** [comment]
Review and let me know

**Comment by Timonwa** [comment]
@lokytech5, the links for the playlist, add the direct link to the playlist and not a link to one of the videos in the playlist.

File: components/backendPageContent/BackendLangs.js [hunk 1]
```diff
@@ -1042,6 +1047,139 @@ const BackendLangs = () => {
...
+                <a target="_blank" href="https://www.youtube.com/watch?v=a-hFbr-4VQQ&list=PLavw5C92dz9Ef4E-1Zi9KfCTXS_IN8gXZ">
+                Roadmap to learning SQL
+                </a>
+              </ResourceLink>
+              <ResourceLink>
+                <a target="_blank" href="https://www.youtube.com/watch?v=vncBSUNb4NA&list=PLEiEAq2VkUUKL3yPbn8yWnatjUg0P0I-Z&index=8">
+                SQL Tutorial For Beginners | Simplilearn
+                </a>
+              </ResourceLink>
+              <ResourceLink>
```
Comment (Timonwa) [inline components/backendPageContent/BackendLangs.js:1099]: Let this link come first

File: components/backendPageContent/BackendLangs.js [hunk 2]
```diff
@@ -1042,6 +1047,139 @@ const BackendLangs = () => {
...
+      <ResourceCardLinks>
+             {/* Others here */}
+             <ResourceLink>
+                <a target="_blank" href="https://www.viget.com/articles/10-sql-tricks-that-i-like/">
+                A list of 10 SQL tricks
+                </a>
+                <ResourceBadge>Article</ResourceBadge>
+              </ResourceLink>
+              <ResourceLink>
+                <a target="_blank" href="https://towardsdatascience.com/10-quick-sql-tips-after-writing-daily-in-sql-for-3-years-37bdba0637d0">
```
Comment (Timonwa) [inline components/backendPageContent/BackendLangs.js:1174]: This article has a paywall. Can you replace it with one that doesn't require payment before accessing it?

**Review by Timonwa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**